pycytominer/operations/transform.py

```diff
@@ -170,7 +170,7 @@ def fit(self, X, y=None):
         # Get the mean of the features (columns) and center if specified
         self.median = X.median()
         self.mad = pd.Series(
-            median_absolute_deviation(X, nan_policy="omit"), index=self.median.index
+            median_abs_deviation(X, nan_policy="omit", scale=1/1.4826), index=self.median.index
```

Comment:
The `scale` param is required to preserve previous behavior. More info at [scipy doc](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.median_absolute_deviation.html#scipy.stats.median_absolute_deviation)